logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -425,12 +473,13 @@ class DummyOutput2 < LogStash::Outputs::DroppingDummyOutput; end
       Thread.abort_on_exception = true
 
       @t = Thread.new do
+        Thread.current.abort_on_exception = true
```

Comment:
actually, I'll remove that - I added it because I has a doubt that an exception was not propagated but it did not change anything.